src/meta/src/hummock/hummock_manager.rs

```diff
@@ -220,9 +220,10 @@ where
             .await?
             .unwrap_or_else(CurrentHummockVersionId::new);
 
-        let mut version_ids: HashSet<_> = HummockVersion::list(self.env.meta_store())
+        let versions = HummockVersion::list(self.env.meta_store()).await?;
+        let mut version_ids: BTreeSet<_> = HummockVersion::list(self.env.meta_store())
```

Comment:
`version_ids` is only used in L238 to check whether we have existing versions. I think we can remove it completely and use `versions` to check.

Reply:
> `version_ids` is only used in L238 to check whether we have existing versions. I think we can remove it completely and use `versions` to check.

fixed